src/engraving/libmscore/chord.cpp

```diff
@@ -1425,34 +1425,37 @@ int Chord::calcMinStemLength()
             outsideStaffOffset = line - staff()->lines(tick()) * 4 + 4;
         }
         minStemLength += (outSidePadding + qMax(noteSidePadding, outsideStaffOffset));
-    }
 
-    if (_hook) {
-        int hookOffset = (_hook->height() - qAbs(_hook->smuflAnchor().y())) / _spatium * 4 - 2;
-        // TODO: when the SMuFL metadata includes a cutout for flags, replace this with that metadata
-        // https://github.com/w3c/smufl/issues/203
-        int cutout = up() ? 6 : 8;
-        if (beams() >= 2) {
-            cutout -= 2;
-        }
-        if (score()->styleB(Sid::useStraightNoteFlags)) {
-            cutout = 0;
-        }
-        if (minStemLength < cutout) {
-            minStemLength = hookOffset;
-        } else {
-            minStemLength += hookOffset - cutout;
+        if (_hook) {
+            double smuflAnchor = _hook->smuflAnchor().y() * (_up ? 1 : -1);
+            int hookOffset = (_hook->height() + smuflAnchor) / _spatium * 4 - 2;
+            // TODO: when the SMuFL metadata includes a cutout for flags, replace this with that metadata
+            // https://github.com/w3c/smufl/issues/203
+            int cutout = up() ? 4 * _relativeMag : 6 * _relativeMag;
+            if (beams() >= 2) {
+                cutout -= 2;
```

Comment:
Is it correct that this is _not_ multiplied by `_relativeMag`?

Reply:
Ah, same explanation as above on this one